hack/chains/kaniko-cosign-verify.sh

```diff
@@ -29,19 +31,67 @@ title "Image digest from task result"
 kubectl get $TASKRUN_NAME -o jsonpath="{.status.taskResults[?(@.name == \"IMAGE_DIGEST\")].value}"
 echo
 
-title "Cosign verify the image"
-# Save the output data to a file so we can look at it later
-# (Actually we could just pipe it to jq because the text goes to stderr I think..?)
-show-then-run "cosign verify --key $SIG_KEY $IMAGE_URL --output-file /tmp/verify.out"
+if $INTERNAL_REGISTRY; then
+  title "Cosign verify the image - using Tekton task"
+  COSIGN_VERIFY_TASK_NAME="cosign-verify-$(openssl rand --hex 10)"
+  oc create -f <(echo "
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: $COSIGN_VERIFY_TASK_NAME
+spec:
+  taskRef:
+    name: cosign-verify
+    bundle: quay.io/zregvart_redhat/appstudio-tasks:2022-03-22-130413-1
```

Comment:
Would we aim to use a different image once the tasks are merged and pushed? Do you want to add a comment about what the plan is?

Reply:
Yeah, that's why this is in draft, once https://github.com/redhat-appstudio/build-definitions/pull/37 is merged, we can use https://quay.io/repository/redhat-appstudio/appstudio-tasks instead.